ExeterTeam25/group-software-project
# Adding map to challenge creation screen

To test:

- Run the server
- Log in to user account that can create challenges
- Go to create challenge screen and click on map to select location. Save challenge
- Return to home and make sure challenge is in the right plae


Details:

Created map in create challenges screen
Added onclick events to map to add markers
Markers now set long and lat fields in creation form

Todo: 
Hide lat and long fields when creating challenge
